src/ntimporters/trello/importer.py

```diff
@@ -0,0 +1,344 @@
+                    api_key={"ApiKeyAuth": nt_auth_token},
+                )
+            ),
+            TrelloClient(app_key, auth_token),
+            team_id,
+        )
+
+    except (ImportException, OpenApiException) as exc:
+        return str(exc)
+    return None
+
+
+def _import_data(nt_client: nt.ApiClient, trello_client, team_id: str):
+    """Import everything from Trello to Nozbe Teams"""
+    limits = nt_limits(nt_client, team_id)
+    projects_api = apis.ProjectsApi(nt_client)
+
+    def _import_project(project: dict):
+        """Import trello project"""
+        project_model = models.Project(
+            id=models.Id16ReadOnly(id16()),
+            name=models.NameAllowEmpty(project.get("name")),
+            team_id=models.Id16(team_id),
+            author_id=models.Id16ReadOnly(id16()),
+            created_at=models.TimestampReadOnly(1),
+            last_event_at=models.TimestampReadOnly(1),
+            color=_map_color(project.get("backgroundTopColor")),
+            description=str(project.get("desc") or ""),
+            is_favorite=project.get("is_fav"),
+            sidebar_position=None if not project.get("is_fav") else 1.0,
+            is_open=True,
+            extra="",
+        )
+        nt_project = projects_api.post_project(strip_readonly(project_model)) or {}
+
+        if not (nt_project_id := str(nt_project.get("id"))):
+            raise ImportException("creating project failed")
+
+        if error := _import_project_sections(
+            nt_client, trello_client, nt_project_id, project, nt_members_by_email(nt_client), limits
+        ):
+            raise ImportException(error)
+
+    nt_projects = [elt.get("id") for elt in projects_api.get_projects() if elt.is_open]
+    if len(trello_projects := trello_client.projects()) + len(nt_projects) > 1000 > 0:
+        raise ImportException("LIMIT projects")
+    for project in trello_projects:
+        if error := _import_project(project):
+            raise ImportException(error)
+
+
+# pylint: disable=too-many-arguments
+def _import_project_sections(
+    nt_client,
+    trello_client,
+    nt_project_id: str,
+    project: dict,
+    nt_members: tuple[dict, str],
+    limits: dict,
+):
+    """Import trello lists as project sections"""
+    nt_api_sections = apis.ProjectSectionsApi(nt_client)
+    nt_api_tasks = apis.TasksApi(nt_client)
+    tags_mapping = _import_tags_per_project(nt_client, trello_client, project, limits)
+
+    def _parse_timestamp(trello_timestamp: Optional[str]) -> Optional[models.TimestampNullable]:
+        """Parses Trello timestamp into NT timestamp format"""
+        if not trello_timestamp:
+            return None
+        return models.TimestampNullable(int(isoparse(trello_timestamp).timestamp() * 1000))
+
+    # import project sections
+    if (
+        len(trello_sections := trello_client.sections(project.get("id")))
+        > limits.get("project_sections", 0)
+        > 0
```

Comment:
```suggestion
        > -1
```
otherwise if `limits["project_sections"] == 0`, condition would resolve to False and error wouldn't be raised